src/Sql/Sql.Test/ScenarioTests/DatabaseBackupTests.ps1

```diff
@@ -494,4 +494,60 @@ function Test-ShortTermRetentionPolicy
+	Assert-AreNotEqual $sourceBackups.Count 0
+	$sourceBackup = $sourceBackups[0]
+
+	# Copy backup
+	$copyBackupResults = Copy-AzSqlDatabaseLongTermRetentionBackup -Location $sourceLocationName -ServerName $sourceBackup.ServerName -DatabaseName $sourceBackup.DatabaseName -BackupName $sourceBackup.BackupName -ResourceGroupName $resourceGroupName -TargetDatabaseName $targetDatabaseName -TargetServerName $TargetServerName -TargetSubscriptionId '01c4ec88-e179-44f7-9eb0-e9719a5087ab' -TargetResourceGroupName $resourceGroupName
+	$targetBackup = Get-AzSqlDatabaseLongTermRetentionBackup -Location $copyBackupResults.TargetLocation -ResourceGroup $copyBackupResults.TargetResourceGroupName -ServerName $copyBackupResults.TargetServerName -DatabaseName $copyBackupResults.TargetDatabaseName -BackupName $copyBackupResults.TargetBackupName
+	Assert-AreEqual $targetDatabaseName $targetBackup.DatabaseName 
+	Assert-AreEqual $targetServerName $targetBackup.ServerName
+}
+
+function Test-SetLongTermRetentionBackup
+{
+	# MANUAL INSTRUCTIONS
+	# Create a server and database and fill in the appropriate information below
+	# Set the weekly retention on the database so that the first backup gets picked up, for example:
+	# Set-AzSqlDatabaseLongTermRetentionPolicy -ResourceGroup $resourceGroup -ServerName $serverName -DatabaseName $databaseName -WeeklyRetention P1W
+	# Wait about 18 hours until it gets properly copied and you see the backup when run get backups, for example:
+	# Get-AzSqlDatabaseLongTermRetentionBackup -Location $locationName -ServerName $serverName -DatabaeName $databaseName -ResourceGroupName $resourceGroup
+	$resourceGroupName = "ayang-rg"
+	$locationName = "southeastasia"
+	$serverName = "ayang-southeastasia"
+	$databaseName = "dbcopysource"
+
+	# Fetch a backup
+	$backups = Get-AzSqlDatabaseLongTermRetentionBackup -Location $locationName -ResourceGroupName $resourceGroupName -ServerName $serverName -DatabaseName $databaseName
+	$backup = $backups[0]
+
+	# Change backup storage redundancy of database, so LTR backup's backup storage redundancy can be changed 
+	# LTR backup's backup storage redundancy must match database's backup storage redundancy
+	# Use a backup storage redundancy different from the CurrentBackupStorageRedundancy value in Get-AzSqlDatabase
+	Set-AzSqlDatabase -DatabaseName $databaseName -ServerName $serverName -ResourceGroupName $resourceGroupName -BackupStorageRedundancy Local
+
+	# Change backup's backup storage redundancy 
+	$backupAfterSet = Set-AzSqlDatabaseLongTermRetentionBackup -Location $locationName -ServerName $backup.ServerName -DatabaseName $backup.DatabaseName -BackupName $backup.BackupName -ResourceGroupName $backup.ResourceGroupName -BackupStorageRedundancy Local
+
+	# Set-AzSqlDatabaseLongTermRetentionBackup returns after target BSR is set
+	Assert-AreEqual $backupAfterSet.RequestedBackupStorageRedundancy $backupAfterSet. BackupStorageRedundancy 
```

Comment:
you have extra space in between? Did your test pass?
$backupAfterSet. BackupStorageRedundancy

Reply:
Space in between is fine functionality-wise, but changing it for syntax